internal/driver/driver.go

```diff
@@ -342,3 +343,9 @@ func valueExtractor(ix int) sampleValueFunc {
 		return v[ix]
 	}
 }
+
+func addPseudoLocs(p *profile.Profile, cfg config, ui plugin.UI) {
+	tr, tl := p.AddPseudoLocs(cfg.TagRoot, cfg.TagLeaf)
```

Comment:
Could we follow the "tag root / leaf" terminology, without introducing the "pseudo locations" speak? E.g. addRootLeafTagLocations.

Reply:
Removed the use of word "pseudo" in  functions and file names.